source/forcing_shf.F90

```diff
@@ -660,7 +660,31 @@ subroutine init_shf(STF)
 !-----------------------------------------------------------------------
 
    case ('analytic')
+      allocate( SHF_DATA(nx_block,ny_block,max_blocks_clinic, &
+                         shf_data_num_fields,1))
 
+      !$OMP PARALLEL DO PRIVATE(iblock)
+      do iblock=1,nblocks_clinic
+         select case (shf_formulation)
+         case ('restoring')
+            SHF_DATA(:,:,iblock,shf_data_sst,1) = &
+               25.0_r8*(c1 - sin(ULAT(:,:,iblock)))
```

Comment:
this changes the analytic case to use 25 instead of 28

Reply:
This should go back to standard value, I didn't corrected that back after creating 'amuse' case

Reply:
fixed